### PR TITLE
[Merged by Bors] - feat(SetTheory/ZFC/Basic): make `sInter` computable

### PR DESCRIPTION
**Pull request by vihdzp**
Moved from #15624 with permission from the original author.

Co-authored by: RustyYato <rustyyato@gmail.com>

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by YaelDillies — APPROVED** [review]
Can you please remove the github ping from the PR description? Else, once this PR is merged, this will ping the person every time the commit is added to a branch.

maintainer merge

**Comment by eric-wieser** [comment]
> Else, once this PR is merged, this will ping the person every time the commit is added to a branch.

... which in normal git operations should never happen! This only happens when the commit is rebased on a branch accidentally

But yes, the convention understood by github is `Co-authored-by: Name <email@site.com>` not `Co-authored-by: @username`.

File: Mathlib/SetTheory/ZFC/Basic.lean
Comment (eric-wieser) [inline Mathlib/SetTheory/ZFC/Basic.lean:893]: I don't agree with "special case" here; maybe "we define" or similar instead.

Comment (vihdzp) [reply Mathlib/SetTheory/ZFC/Basic.lean]: I've reworded using "define".

**Comment by eric-wieser** [comment]
My comment about `Co-authored-by` still applied, so I did it myself

bors merge

**Comment by eric-wieser** [comment]
Nevermind, build is failing

bors d+

**Comment by vihdzp** [comment]
The build wasn't actually failing, there was just some problem with connecting to some server.

bors r+

**Pull request closed without merging**
